doc/chapters/gateways/index.rst

```diff
@@ -268,23 +289,24 @@ An example configuration can look like this::
         "env-var-suffix": "some-path-suffix", // define a suffix for the path set in the environment variable defined by "env-var-name"
     }
```

Comment:
Not valid config. All gateway configurations needs to be formated as JSON lists.